include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -514,7 +514,11 @@ The data will be distributed between threads as follows:
 
   let parameters = (
     ins
-    ArrayRefParameter<"unsigned">:$warpsPerCTA
+    ArrayRefParameter<"unsigned">:$warpsPerCTA,
```

Comment:
This commits adds 'order' field for mfma layout as well. However we still don't use it, since it's not required for simple chain dot, but only for FA where we need result of second dot in transposed layout. I will probably move adding 'order' in mfma layout to separate PR.